packages/home-spa/.env.example

```
@@ -1 +1 @@
-API_URL=
\ No newline at end of file
+API_URL=https://localhost:8080/graphql
```

Why? I'd suggest keeping this:
```suggestion
API_URL=
```

---

Noted and changd